slicedimage/backends/_base.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+
+class Backend(object):
```

Comment:
I'm wondering if the `NotImplemented` methods should be private.

Reply:
Can't do protected methods in python.

Reply:
Sorry, meant soft-private `_read_file_handle_callable()` because it seemed like they only get used within the class, and aren't called by outside methods.

Reply:
I have not traditionally used _xxx for "abstract" methods, because PyCharm considers that a violation of abstraction.  However, I can see the advantage of doing it this way.

Reply:
Huh, interesting. Ok. 👍

Reply:
Personally, I am a fan of the object models of java/kotlin/swift/etc. :)

Reply:
It turns out the abstract methods are sometimes used outside of the module.